lib/commandFactory.js

```diff
@@ -70,20 +70,24 @@ class CommandFactory extends BaseFactory {
                 if (latestCommand) {
                     latestCommand.latest = false;
 
-                    return latestCommand.update().then(() => latestCommand.version);
+                    return latestCommand.update().then(() => latestCommand);
                 }
 
                 return null;
             })
-            .then(latestVersion => {
-                if (!latestVersion) {
+            .then(latestCommand => {
+                if (!latestCommand) {
                     config.version = minor ? `${major}${minor}.0` : `${major}.0.0`;
+                    config.trusted = false;
                 } else {
                     // eslint-disable-next-line max-len
-                    const [, latestMajor, latestMinor, latestPatch] = VERSION_REGEX.exec(latestVersion);
+                    const [, latestMajor, latestMinor, latestPatch] = VERSION_REGEX.exec(latestCommand.version);
                     const patch = parseInt(latestPatch.slice(1), 10) + 1;
 
                     config.version = `${latestMajor}${latestMinor}.${patch}`;
+
+                    // eslint-disable-next-line no-unneeded-ternary
+                    config.trusted = latestCommand.trusted ? true : false;
```

Comment:
won't `config.trusted = latestCommand.trusted` ? work ?

Reply:
`trusted` could be null, so just explicitly setting it to false